accounts.go

```diff
@@ -7,21 +7,28 @@ import (
 
 // Account represents an individual account on your site
 type Account struct {
-	XMLName          xml.Name `xml:"account"`
-	Code             string   `xml:"account_code,omitempty"`
-	State            string   `xml:"state,omitempty"`
-	Username         string   `xml:"username,omitempty"`
-	Email            string   `xml:"email,omitempty"`
-	FirstName        string   `xml:"first_name,omitempty"`
-	LastName         string   `xml:"last_name,omitempty"`
-	CompanyName      string   `xml:"company_name,omitempty"`
-	VATNumber        string   `xml:"vat_number,omitempty"`
-	TaxExempt        NullBool `xml:"tax_exempt,omitempty"`
-	BillingInfo      *Billing `xml:"billing_info,omitempty"`
-	Address          Address  `xml:"address,omitempty"`
-	AcceptLanguage   string   `xml:"accept_language,omitempty"`
-	HostedLoginToken string   `xml:"hosted_login_token,omitempty"`
-	CreatedAt        NullTime `xml:"created_at,omitempty"`
+	XMLName                 xml.Name `xml:"account"`
+	Code                    string   `xml:"account_code,omitempty"`
+	State                   string   `xml:"state,omitempty"`
+	Username                string   `xml:"username,omitempty"`
+	Email                   string   `xml:"email,omitempty"`
+	FirstName               string   `xml:"first_name,omitempty"`
+	LastName                string   `xml:"last_name,omitempty"`
+	CompanyName             string   `xml:"company_name,omitempty"`
+	VATNumber               string   `xml:"vat_number,omitempty"`
+	TaxExempt               NullBool `xml:"tax_exempt,omitempty"`
+	BillingInfo             *Billing `xml:"billing_info,omitempty"`
+	Address                 Address  `xml:"address,omitempty"`
+	AcceptLanguage          string   `xml:"accept_language,omitempty"`
+	HostedLoginToken        string   `xml:"hosted_login_token,omitempty"`
+	CreatedAt               NullTime `xml:"created_at,omitempty"`
+	UpdatedAt               NullTime `xml:"updated_at,omitempty"`
+	ClosedAt                NullTime `xml:"closed_at,omitempty"`
+	HasLiveSubscription     NullBool `xml:"has_live_subscription,omitempty"`
```

Comment:
I was unable to find the fields in lines 27-31 in the documentation or in test API calls. Please provide steps to recreate a get accounts call where these are present.

Reply:
Strange, I just tested the endpoint again and also logged the body of the response. Everything looks okay to me.

Go code
```go
client := recurly.NewClient(config.RecurlySubdomain, config.RecurlyPrivateAPIKey, nil)
resp, a, err := client.Accounts.Get("uonmaufkatyltuoheiagjimqokgfkhjoitewkupw")
assert.NoError(t, err)

t.Logf("a: %#+v\n", a)
```

Console
```
a: &recurly.Account{XMLName:xml.Name{Space:"", Local:"account"}, Code:"uonmaufkatyltuoheiagjimqokgfkhjoitewkupw", State:"active", Username:"0", Email:"chloejackson641@test.net", FirstName:"", LastName:"", CompanyName:"Maskpolar", VATNumber:"", TaxExempt:recurly.NullBool{Bool:false, Valid:false}, BillingInfo:(*recurly.Billing)(nil), Address:recurly.Address{Address:"", Address2:"", City:"", State:"", Zip:"", Country:"", Phone:""}, AcceptLanguage:"", HostedLoginToken:"vWKfsWhjXLWmpALeJqd4G4Fv1P15sxiJ", CreatedAt:recurly.NullTime{Time:(*time.Time)(0xc4203574c0), Raw:""}, UpdatedAt:recurly.NullTime{Time:(*time.Time)(0xc4203575e0), Raw:""}, ClosedAt:recurly.NullTime{Time:(*time.Time)(nil), Raw:""}, HasLiveSubscription:recurly.NullBool{Bool:false, Valid:true}, HasActiveSubscription:recurly.NullBool{Bool:false, Valid:true}, HasFutureSubscription:recurly.NullBool{Bool:false, Valid:true}, HasCanceledSubscription:recurly.NullBool{Bool:false, Valid:true}, HasPastDueInvoice:recurly.NullBool{Bool:false, Valid:true}}
```

Body
```xml
<?xml version="1.0" encoding="UTF-8"?>
<account href="https://test.recurly.com/v2/accounts/uonmaufkatyltuoheiagjimqokgfkhjoitewkupw">
  <adjustments href="https://test.recurly.com/v2/accounts/uonmaufkatyltuoheiagjimqokgfkhjoitewkupw/adjustments"/>
  <account_balance href="https://test.recurly.com/v2/accounts/uonmaufkatyltuoheiagjimqokgfkhjoitewkupw/balance"/>
  <invoices href="https://test.recurly.com/v2/accounts/uonmaufkatyltuoheiagjimqokgfkhjoitewkupw/invoices"/>
  <shipping_addresses href="https://test.recurly.com/v2/accounts/uonmaufkatyltuoheiagjimqokgfkhjoitewkupw/shipping_addresses"/>
  <subscriptions href="https://test.recurly.com/v2/accounts/uonmaufkatyltuoheiagjimqokgfkhjoitewkupw/subscriptions"/>
  <transactions href="https://test.recurly.com/v2/accounts/uonmaufkatyltuoheiagjimqokgfkhjoitewkupw/transactions"/>
  <account_code>uonmaufkatyltuoheiagjimqokgfkhjoitewkupw</account_code>
  <state>active</state>
  <username>0</username>
  <email>chloejackson641@test.net</email>
  <cc_emails nil="nil"></cc_emails>
  <first_name nil="nil"></first_name>
  <last_name nil="nil"></last_name>
  <company_name>Maskpolar</company_name>
  <vat_number nil="nil"></vat_number>
  <address>
    <address1 nil="nil"></address1>
    <address2 nil="nil"></address2>
    <city nil="nil"></city>
    <state nil="nil"></state>
    <zip nil="nil"></zip>
    <country nil="nil"></country>
    <phone nil="nil"></phone>
  </address>
  <accept_language nil="nil"></accept_language>
  <hosted_login_token>vWKfsWhjXLWmpALeJqd4G4Fv1P15sxiJ</hosted_login_token>
  <created_at type="datetime">2017-09-01T16:22:27Z</created_at>
  <updated_at type="datetime">2017-09-01T16:22:27Z</updated_at>
  <closed_at nil="nil"></closed_at>
  <has_live_subscription type="boolean">false</has_live_subscription>
  <has_active_subscription type="boolean">false</has_active_subscription>
  <has_future_subscription type="boolean">false</has_future_subscription>
  <has_canceled_subscription type="boolean">false</has_canceled_subscription>
  <has_past_due_invoice type="boolean">false</has_past_due_invoice>
</account>
```

The Recurly account I use is pretty new. Maybe it has something to do with that?

Also, under [API Credentials](https://hundred5-test.recurly.com/developer/api_keys) it states that I'm accessing API version 2.5

<img width="285" alt="screen shot 2017-09-01 at pm 6 32 04" src="https://user-images.githubusercontent.com/14206503/29979153-a6eb0542-8f44-11e7-9b02-f59b9e56d6a3.png">

Furthermore, the [API Docs](https://dev.recurly.com/docs/account-object) also include these fields.

Conclusion: Looks like these fields were added in versions 2.2 and 2.5